### DISCUSSION TITLE
I can't seem to be able to fish enchanted items.

### DISCUSSION THREAD

**Discussion by absurdother** [MrKinau/FishingBot#145] [Q&A] [answered]
![fishing](https://user-images.githubusercontent.com/31885693/127747193-e25e3e7c-6374-497b-bf1e-3d131c362173.png)

I can't seem to be able to fish enchanted items. I've been using the last version of FisherBot and, although a lot of fish, trash and other items do come, I do not receive any treasure whatsoever (even when fishing 1000+ items). I've also tried using different fishing rods, some with Luck Of The Sea II or III, and Lure II and III; and yet these are apparently not fishing a single enchanted item/book at all.

I've fished over 2000+ items but no enchanted one already, and I don't think it's just the normal ratio since I've used many enchanted fishing rods; I'm still trying it though (I've used /stuck and such, doesn't work) and will update if I get an enchanted item, but this is weird.

**Comment by Mottekmalocher** [comment, marked as answer]
To catch items in the treasure category, the bobber must be in open water. This is technically defined as the 5×4×5 vicinity around the bobber resting on the water surface (2 blocks away horizontally, 2 blocks above the water surface, and 2 blocks deep), and the entire 5×5 surface must be exposed to the sky regardless of the elevation of the volume of water.

https://minecraft.fandom.com/wiki/Fishing#Junk_and_treasure

**Reply by absurdother** [reply]
Thanks, you may add this to some Q&A or anything, the solution was perfect. I've adjusted my fishing pond and it's meeting requirements now and fishing like crazy. Awesome!
